Dynatrace/dynatrace-operator
# Update build and dev dependencies in makefile

# Description
As dependabot  can't update these, we have to do it manually

## How can this be tested?
Check that those versions are what you are using locally, and that all linting and build commands work


## Checklist
